### PR TITLE
[WIP] Create authentication front-end prototype.

### PR DESCRIPTION
**Pull request by LightFelicis**

### CODE REVIEW

**Comment by LightFelicis** [comment]
I created a draft of authentication process. Currently logging in is done by mock authenticator (having 'admin' cookie grants admin privileges, 'user' cookie grants user privileges).

File: src/components/Authenticator.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import Cookies from 'js-cookie';
+
+const mockAuth = {
+  isAdmin: function() {
+    let adminCookie = Cookies.get('admin');
```
Comment (xmxwx) [inline src/components/Authenticator.js:5]: nit: Can you use `const` here?

File: src/components/landingPage/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  const classes = useStyles();
+
+  /* This part will be fetched from server */
+  let landingPageContent = [
+    {to: '/self-service', label: 'Self service page'},
+    {to: '/restricted-area', label: 'Restricted area'}];
+  if (mockAuth.isAdmin()) {
+    landingPageContent.push({to: '/admin', label: 'Admin panel'});
+  }
+  /* End of part fetched from the server */
```
Comment (xmxwx) [inline src/components/landingPage/LandingPage.js:19]: nit: End of part *to be* fetched from the server

**Review by xmxwx — APPROVED** [review]

File: src/components/landingPage/LandingPage.js (hunk 2 above)
Comment (LightFelicis) [reply src/components/landingPage/LandingPage.js]: Done.

File: src/components/Authenticator.js (hunk 1 above)
Comment (LightFelicis) [reply src/components/Authenticator.js]: Done.

**Review by leon-kastler-google — APPROVED** [review]
two styling things, otherwise looks great!

File: src/index.js [hunk 3]
```diff
@@ -5,10 +5,10 @@ import App from './App';
...
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
+    <React.StrictMode>
+      <App/>
+    </React.StrictMode>,
+    document.getElementById('root'),
```
[the remark this answers is not in the document]
Comment (LightFelicis) [reply src/index.js:11]: This file was automatically generated file by create-react-app.
My formatter was following the rule: https://google.github.io/styleguide/jsguide.html#formatting-indent

[the remark this answers is not in the document]
Comment (LightFelicis) [reply src/index.js:11]: The standard indentation is still 2 spaces, but when continuing new line (arguments of ReactDOM.render())
next line is indented by 4 spaces.

Does it make sense?

[the remark this answers is not in the document]
Comment (leon-kastler-google) [reply src/index.js:11]: maybe :-)

File: src/App.js [hunk 4]
```diff
@@ -1,46 +1,82 @@
...
+      <Router>
+        <div>
+          <Switch>
+
+            <PrivateRoute
+                path="/"
+                exact
+                authFunction={() => {
+                  return (!mockAuth.isUser());
+                }}
```
[the remark this answers is not in the document]
Comment (LightFelicis) [reply src/App.js]: Done, thanks! :)

**Pull request merged**